modules/ROOT/pages/dataweave-formats-flatfile.adoc

```diff
@@ -23,17 +26,91 @@ output application/flatfile schemaPath="src/main/resources/test-data/QBReqRsp.es
 payload
 ----
 
+// CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
+
+
+[[properties]]
+== Configuration Properties
+
+DataWeave supports the following configuration properties for this format.
+
+
+[[reader_properties]]
+=== Reader Properties
+
+This format accepts properties that provide instructions for reading input data.
+
+
+[cols="1,1,1,3a", options="header"]
+|===
+|Parameter |Type |Default|Description
+|`enforceRequires`  |`Boolean`|`false`|Produces an error when set to `true` if a required value is missing. Valid values are `true` or `false`.
+
+Valid values are `true` or `false`.
+|`missingValues`  |`String`|`null`|Specifies a fill character that represents missing values.
```

Comment:
TODO: Default differs by format

"nulls for copybook schema, spaces otherwise"

Looks like we can't share  between formats.

Reply:
and `null` is incorrect regardless.